cutlass/cf.go

```diff
@@ -415,7 +416,12 @@ func (a *App) Get(path string, headers map[string]string) (string, map[string][]
 	if err != nil {
 		return "", map[string][]string{}, err
 	}
-	client := &http.Client{}
+	_, insecureSkipVerify := os.LookupEnv("CUTLASS_SKIP_TLS_VERIFY")
```

Comment:
@drnic Perhaps it should verify what the env is set to, instead of just checking when it's set?

Reply:
@dfreilich fair 'nuf :) Fixed.